database/factories/GroupFactory.php

```diff
@@ -21,7 +21,6 @@ public function definition(): array
         return [
             'name' => $this->faker->word(),
             'event_id' => Event::all(['id'])->random(),
-            'course_id' => Course::all(['id'])->random(),
```

Comment:
Why was this removed?

Reply:
There is no course_id anymore, see PR #673